Makefile

```diff
@@ -193,6 +194,11 @@ aevm-test: aevm-test-deps
 	@./rebar3 eunit --application=aevm
 
 aevm-test-deps: $(AEVM_EXTERNAL_TEST_DIR)/ethereum_tests
+aevm-test-deps:
+	$(eval VER=$(shell git -C "$(AEVM_EXTERNAL_TEST_DIR)/ethereum_tests" log -1 --pretty=format:"%H"))
```

Comment:
`git rev-parse HEAD`?

Reply:
As this is a stopgap, I'll not update. But indeed. That might have been more clear.